app/main.py

```diff
@@ -1 +1,17 @@
-# write your code here
+from app.cafe import Cafe
+from app.errors import VaccineError, NotWearingMaskError
+
+
+def go_to_cafe(friends: list, cafe: Cafe) -> str:
```

Comment:
It's recommended to use type hinting for the `cafe` parameter as well. This will make the function signature more informative and clear. For example: `def go_to_cafe(friends: list, cafe: Cafe) -> str:`